tests/test_easy.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+from dotenv import load_dotenv
+import pytest
+from pydantic import BaseModel, Field
+
+load_dotenv() # FIXME: probably don't want this here.
+
```

Comment:
Rather than `load_dotenv()` and using real APIs, you can use the `MockProvider` (see below):

```python
from lasagna.mock_provider import (
    MockProvider,
)

...

@pytest.mark.asyncio
async def test_thing():
    binder = bind_model(MockProvider, 'some_model')
    ...
```

Reply:
It returns deterministic "stuff" so you can assert the results on it as well.

Reply:
I won't look at the rest of this file for now, since I figure it will change a bunch after you use the MockProvider.

Reply:
Will mock provider generate the structured output required to pass the tests?